README.md

```diff
@@ -173,29 +162,8 @@ import (
 )
 
 func main() {
-    directCfg := &wavefront.DirectConfiguration {
-        Server : "https://INSTANCE.wavefront.com", // your Wavefront instance URL
-        Token : "YOUR_API_TOKEN",                  // API token with direct ingestion permission
-
-        // Optional configuration properties. Default values should suffice for most use cases.
-        // override the defaults only if you wish to set higher values.
-
-        // max batch of data sent per flush interval. defaults to 10,000.
-        // recommended not to exceed 40,000.
-        BatchSize : 10000,
-
-        // size of internal buffer beyond which received data is dropped.
-        // helps with handling brief increases in data and buffering on errors.
-        // separate buffers are maintained per data type (metrics, spans and distributions)
-        // defaults to 500,000. higher values could use more memory.
-        MaxBufferSize : 500000,
-
-        // interval (in seconds) at which to flush data to Wavefront. defaults to 1 Second.
-        // together with batch size controls the max theoretical throughput of the sender.
-        FlushIntervalSeconds : 1,
-    }
-
-    sender, err := wavefront.NewDirectSender(directCfg)
+    // 11111111-2222-3333-4444-555555555555 is the API token
+    sender, err := wavefront.NewSender("https://11111111-2222-3333-4444-555555555555@surf.wavefront.com")
```

Comment:
should we put a place holder like {INSTANCE} instead of 11111111-2222-3333-4444-555555555555..
Or is there a cluster named 11111111-2222-3333-4444-555555555555? :)

Reply:
Thanks Shavi. This long string is actually the API token. I wanted to make it clear to developers that they must include it here without using a real API token.